index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- Favicons -->
+    <link rel="shortcut icon" href="favicon.ico">
```

Comment:
Where is favicon added?

Reply:
I have added favicon.ico in root directory, please check.

Reply:
I cannot see that in this PR. Can you please push your changes?